hilti/runtime/include/types/integer.h

```diff
@@ -158,7 +158,7 @@ inline Result<std::tuple<integer::safe<T>, D>> unpack(D b, ByteOrder fmt) {
         return result::Error("insufficient data to unpack integer");
 
     uint8_t raw[sizeof(T)];
-    b = b.extract(raw);
+    b = b.extract(raw, sizeof(raw));
```

Comment:
Let's be a little bit more verbose to make 110% clear we want the number of elements:

```suggestion
    b = b.extract(raw, sizeof(raw)/sizeof(raw[0]));
```

Reply:
I'll stay with just `sizeof` and will use that in the tests, too. I think that's pretty clear.